### PR TITLE
Fixing an issue where the provider uses a Name instead of ProviderName

### PR DESCRIPTION
**Pull request by darkbushido**
Expanding the responses hashes in the ccp_request_spec.rb
The mocks were hiding the errors

department-of-veterans-affairs/va.gov-team#6010

## Description of change
PPMS `v1.0/ProviderLocator` Endpoint returns a hash with the following keys
`CareSite` and `ProviderName`

PPMS `v1.0/Providers(1407842941)` Endpoint returns a hash with the following key
`Name`

`v1.0/ProviderLocator` uses `CareSite` and `ProviderName`
`v1.0/Providers(1407842941)` uses `Name`

With the provider name change pr #3902, it didn't look like anything was using `Name`
This is not the case and this PR addresses the issue.

## Acceptance Criteria (Definition of Done)
 - [x] Facility or Provider name should appear above Facility type text on the Detail page
 - [x] The Detail page's title should match the provider name

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: app/serializers/provider_serializer.rb [hunk 1]
```diff
@@ -13,17 +13,22 @@ def unique_id
...
   end
 
+  def fipper_enabled_name
+    possible_name = case object.ProviderType
+                    when /GroupPracticeOrAgency/i
+                      object.CareSite
+                    else
+                      object.ProviderName
+                    end
+    [possible_name, object.Name].find(&:present?)
```
Comment (omgitsbillryan) [inline app/serializers/provider_serializer.rb]: I find myself getting tripped up on this line - is this equivalent to this?

```ruby
possible_name || object.Name
```

**Comment by omgitsbillryan** [comment]
Mind being more verbose in the OP description. I don't understand this:

>PPMS Response arrays use ProviderName
Single Responses use Name
For some reason Provider was getting Name

File: app/serializers/provider_serializer.rb (hunk 1 above)
Comment (darkbushido) [reply app/serializers/provider_serializer.rb]: it is, `possible_name || object.Name` was causing an issue with codeclimate

Comment (omgitsbillryan) [reply app/serializers/provider_serializer.rb]: Gotcha, weird, but they're functionally equivelant, yes? Suppose I'm just asking for my own curiosity.

Comment (omgitsbillryan) [reply app/serializers/provider_serializer.rb]: Also weird that codeclimate seems to be OK with line 18: `object.ProviderName || object.Name`, but not here? 🤔

**Pull request merged**